### PR TITLE
Add CHANGELOG for v2.2.0

### PR DESCRIPTION
**Pull request by jsafrane**
/kind documentation

/assign @msau42 

Not sure about min Kubernetes version - now the attacher requires CSINodeInfo. It was beta in 1.14 and GA in 1.17. Can we assume it's enabled in 1.15 or should we raise it to 1.17?

```release-note
NONE
```

### CODE REVIEW

File: CHANGELOG-2.2.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Changelog since v2.1.0
+
+### Bug Fixes
+
+- A bug that prevented the external-attacher from releasing its finalizer from a `PersistentVolume` object that was created using a legacy storage class provisioner has been fixed. ([#218](https://github.com/kubernetes-csi/external-attacher/pull/218), [@rfranzke](https://github.com/rfranzke))
```
Comment (msau42) [inline CHANGELOG-2.2.md]: prefix that it is related to the csi migration feature to make it more obvious

File: README.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ This information reflects the head of this branch.
 
 | Compatible with CSI Version                                                                | Container Image             | Min K8s Version | Recommended K8s Version |
 | ------------------------------------------------------------------------------------------ | ----------------------------| --------------- | ----------------------- |
-| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | quay.io/k8scsi/csi-attacher | 1.15            | 1.17                    |
+| [CSI Spec v1.2.0](https://github.com/container-storage-interface/spec/releases/tag/v1.2.0) | quay.io/k8scsi/csi-attacher | 1.15            | 1.18                    |
```
Comment (msau42) [inline README.md]: all our release notes and documentation say min is 1.14. Do you know why it's 1.15 here?

File: CHANGELOG-2.2.md (hunk 1 above)
Comment (jsafrane) [reply CHANGELOG-2.2.md:5]: Added "... using a legacy storage class provisioner and *migrated to CSI*"

File: README.md (hunk 2 above)
Comment (jsafrane) [reply README.md:24]: It was added in https://github.com/kubernetes-csi/external-attacher/pull/155, without any obvious reason (at least I don't see it in changelog).

Comment (jsafrane) [reply README.md:24]: Fixed to 1.14

**Pull request merged**